src/components/tag-list/index.tsx

```diff
@@ -0,0 +1,18 @@
+import Tag, { TagProps } from "../tag";
+import s from "./TagList.module.css";
+
+interface TagListProps {
+  tags: TagProps[];
+}
+
+export default function TagList({ tags }: TagListProps) {
+  return (
+    <ul className={s.tagList}>
+      {tags.map((tag, index) => (
+        <li>
```

Comment:
You'll need a key prop here!

Reply:
serves me right for moving it away from the component 😂, should be all sorted now 🙏